README.md

```diff
@@ -26,6 +26,24 @@ under that directory will contain instructions on how to run the sample.
 
 If desired, you can easily use the Maven Archetype from a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide). The generated starter code is simply Maven projects. You can easily load, explore and run the code in a Maven capable IDE such as [Eclipse](https://www.eclipse.org/ide).
 
+## Instructions On How To Use Archetypes From An IDE Such As Eclipse
+Install Maven:
+Before you can use Maven in Eclipse, you need to install it on your system. You can download Maven from the official website (https://maven.apache.org/download.cgi) and follow the installation instructions.
+
+Install the Maven Integration for Eclipse:
```

Comment:
This should not be needed. For Jakarta EE projects, we should specify the Enterprise version of Eclipse.